<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefEditDialog</class>
 <widget class="QDialog" name="RefEditDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Reference</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="searchHBox">
     <item>
      <widget class="SearchLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
       <property name="icon">
        <iconset resource="../../../libwar/resources/images.qrc">
         <normaloff>:/icons/edit-find.png</normaloff>:/icons/edit-find.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="refList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="nameHBox">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="WarButtonBox" name="saveBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WarButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header location="global">widgets/WarButtonBox.h</header>
  </customwidget>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/SearchLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../libwar/resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
